build/webpack.config.babel.js

```diff
@@ -5,10 +5,17 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
 const { CleanWebpackPlugin } = require('clean-webpack-plugin')
 const CopyPlugin = require('copy-webpack-plugin')
 const babelConfig = require('../.babelrc.js')
-const getVersion = require('git-repo-version')
+const fs = require('fs')
+
+const SHORT_COMMIT_SHA_LENGTH = 8
+
+const readVersionName = () => {
+  const versionFile = fs.readFileSync(path.resolve(__dirname, '../version.json'))
+  return JSON.parse(versionFile).versionName
+}
 
 const createConfig = (env = {}) => {
-  const { config_name: buildConfigName, production, debug } = env
+  const { config_name: buildConfigName, production, debug, commit_sha: commitSha, version_name: versionName } = env
```

Comment:
Where is `commitSha` coming from?

Reply:
Supplied by CircleCI. It is not necessary though.